lavalink/Client.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, bot, log_level=logging.INFO, loop=asyncio.get_event_loop(), h
         self.ws = WebSocket(
             self, host, password, ws_port, ws_retry, shard_count
         )
-        self.players = PlayerManager(bot)
+        self.players = PlayerManager(bot, self)

```

Comment:
I'd opt for just passing `self` instead - bot can be accessed through `lavalink._bot` if needed